user_tasks/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from django.dispatch import Signal
 
-__version__ = '1.3.0'
+__version__ = '2.0.0'
```

Comment:
Is the version bump (and matching changelog entry) actually needed?  None of the modified files are included in the released package.